config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   root 'home_page#index'
   resources :home_page
+
+  get 'creatives/index'
```

Comment:
It's hard to suggest removing this endpoint because a lot of work was done. All in all, we don't need it in our app. It isn't a part of our business logic so I propose getting rid of it.